cmd/qat_plugin/README.md

```diff
@@ -148,6 +148,9 @@ When using the operator for deploying the plugin with provisioning config, use `
 
 There's also a possibility for a node specific congfiguration through passing a nodename via `NODE_NAME` into initcontainer's environment and passing a node specific profile (`qat-$NODE_NAME.conf`) via ConfigMap volume mount.
 
+CR annotations in [deviceplugin_v1_qatdeviceplugin.yaml](../../deployments/operator/samples/deviceplugin_v1_qatdeviceplugin.yaml) propagate to the DaemonSet annotations. In case AppArmor "unconfined" annotation needs to be dropped in the operator deployments (enabled by default), manually drop it from the CR sample.
```

Comment:
I think the CR annotation does not have to be set but it is only needed if an override is needed:
```suggestion
CR annotations in [deviceplugin_v1_qatdeviceplugin.yaml](../../deployments/operator/samples/deviceplugin_v1_qatdeviceplugin.yaml) propagate to the DaemonSet annotations. By default, the operator based deployment sets AppArmor policy to  `"unconfined"` but this can be overridden by setting the AppArmor annotation to a new value in the CR annotations.
```

Reply:
@mythi
Thanks, makes sense, updated.